node/src/components/consensus/highway_core/state/index_panorama.rs

```diff
@@ -14,19 +14,20 @@ use crate::components::consensus::{
 pub(crate) type IndexPanorama = ValidatorMap<IndexObservation>;
 
 /// The observed behavior of a validator at some point in time.
-#[derive(Clone, DataSize, Eq, PartialEq, Serialize, Deserialize, Hash)]
+#[derive(Clone, Copy, DataSize, Eq, PartialEq, Serialize, Deserialize, Hash)]
 pub(crate) enum IndexObservation {
-    None,
+    /// We have evidence that the validator is faulty.
     Faulty,
-    Correct(u64),
+    /// We have that number of units from the validator.
+    /// That is also the sequence number of the first unit we are still missing.
```

Comment:
Since we're counting from `1` and not `0`, then the sequence number of the first missing unit will be 1 higher than the number of units from that validator we have.

Reply:
No, we're counting from 0: The first unit has sequence number 0. If our latest unit is number 2, that means we have 3 units, and the first we are missing is also 3.

Reply:
Ah, for some reason I rememberred we're counting from 1.